core/app/models/spree/exports/products.rb

```diff
@@ -24,7 +24,27 @@ def scope
       end
 
       def csv_headers
-        @csv_headers ||= Spree::CSV::ProductVariantPresenter::CSV_HEADERS + properties_headers
+        @csv_headers ||= Spree::CSV::ProductVariantPresenter::CSV_HEADERS + taxons_headers + properties_headers
+      end
+
+      def csv_options
+        {
+          taxons_count: max_taxons_count,
+        }
+      end
+
+      def max_taxons_count
+        @max_taxons_count ||= Spree::Taxon.manual
```

Comment:
@Vegann this should per store, not the current collection of records, so it will be the same for all exports